charts/bridgeai-prediction-service/Chart.yaml

```diff
@@ -19,10 +19,21 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.0
+version: 0.1.2
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
 appVersion: "1.16.0"
```

Comment:
```suggestion
# renovate: image=digicatapult/bridgeai-prediction-service
appVersion: "0.1.2"
```

Reply:
This should correspond to the image version of the main application container, the Renovate line is to tell renovate which image it needs to use to automatically update this tag.